core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,670 @@
+          <button type="submit" (click)="onClickCreateLessonButton()"
+                  class="btn oppia-about-button oppia-about-button-browse"
+                  [innerHTML]="'I18N_ACTION_CREATE_LESSON_BUTTON' | translate">
+          </button>
         </div>
       </div>
     </div>
   </div>
-  <div class="oppia-footer-padding-below-banner">
+  <div class="oppia-about-page-section-seven oppia-about-section">
+    <h2 class="oppia-get-started-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_SECTION_SEVEN_TITLE' | translate"></h2>
+    <div class="oppia-about-classroom-button-container-one">
+      <button type="submit" (click)="onClickVisitClassroomButton()"
+              class="btn oppia-about-button oppia-about-button-browse"
+              [innerHTML]="'I18N_ACTION_EXPLORE_LESSONS' | translate">
+      </button>
+    </div>
   </div>
 </div>
+
 <style>
-  .oppia-about-page-container .oppia-about-foundation-tab-anchor {
-    background-color: #00645c;
-    border-radius: 4px;
-    color: white;
-    letter-spacing: 0.7px;
-    width: 33%;
-  }
-  .oppia-about-page-container .oppia-about-foundation-tab-anchor:hover {
-    background-color: #3d9991;
-    text-decoration: none;
-  }
-  .oppia-about-page-container .oppia-about-text-align {
+  .oppia-about-section {
+    background-color: #fff;
+    clear: both;
+    display: flex;
+    justify-content: center;
+    margin: 0 auto;
+    max-width: 100%;
+  }
+  .oppia-about-fixed-section {
+    justify-content: center;
+    width: 1440px;
+  }
+  .oppia-about-page-section-one {
+    background-color: #F4F4F4;
+    height: 60vh;
+    padding-top: 0;
+  }
+  .oppia-about-page-section-one .oppia-about-page-section-one-right {
+    float: right;
+    height: 60vh;
+    width: 50%;
+  }
+  .oppia-about-page-section-one .oppia-about-page-section-one-left {
+    float: right;
+    width: 635px;
+  }
+  .oppia-about-page-section-one .oppia-about-title {
+    font-size: 45px;
+    padding-top: 10vh;
+  }
+  .oppia-about-page-section-one .oppia-about-content {
+    font-size: 23px;
+    font-weight: 400;
+    padding-right: 4vw;
+  }
+  .oppia-about-page-section-one .oppia-about-study-image {
+    float: right;
+  }
+  .oppia-about-page-section-one .oppia-about-study-image img {
+    height: 60vh;
+  }
+  .oppia-about-page-section-two {
+    height: auto;
+    padding-bottom: 5vh;
+    width: 100%;
+  }
+  .oppia-about-page-section-two .oppia-section-two-content {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+  }
+  .oppia-about-page-section-two .oppia-about-title {
+    padding-top: 6.5vh;
     text-align: center;
   }
-  .oppia-about-page-container .oppia-about-three-cols-layout {
-    display: inline-block;
-    margin-right: 20px;
-    width: -webkit-calc((100% - 60px) / 3);
-    width: -moz-calc((100% - 60px) / 3);
-    width: -o-calc((100% - 60px) / 3);
-    width: calc((100% - 60px) / 3);
+  .about-page .oppia-about-page-title {
```

Comment:
Can we just have oppia-about-page? (the other about-page makes it confusing!)

Reply:
changed as section instead of page